src/borg/archiver/debug_cmd.py

```diff
@@ -285,12 +284,7 @@ def do_debug_parse_obj(self, args, repository, manifest):
             cdata = f.read()
 
         repo_objs = manifest.repo_objs
-
-        try:
-            meta, data = repo_objs.parse(id=id, cdata=cdata, decompress=True)
-        except Exception:
-            traceback.print_exc()
-            return EXIT_ERROR
+        meta, data = repo_objs.parse(id=id, cdata=cdata, decompress=True)
```

Comment:
is decompress=True the default value?

if so, leave it away (consistency: you also do not give compress=True in parse-obj).